### PR TITLE
Fix build stuff

### PR DESCRIPTION
**Pull request by TomWoodward**
for: openstax/unified#1972

### CODE REVIEW

File: src/app/errors/components/OuterErrorBoundary.tsx [hunk 1]
```diff
@@ -18,11 +18,13 @@ const OuterErrorBoundary = (props: { children?: React.ReactNode }) => {
       setIntl(intlObject);
     };
 
-    setUpIntl();
-  }, []);
+    if (!intl) {
+      setUpIntl();
```
Comment (Dantemss) [inline src/app/errors/components/OuterErrorBoundary.tsx:22]: OK with no await here?

File: src/app/messages/MessageProvider.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ const MessageProvider = (props: { children?: React.ReactNode }) => {
     };
 
     setUpIntl();
```
Comment (Dantemss) [inline src/app/messages/MessageProvider.tsx:27]: I guess this one doesn't await either so it's probably ok?

File: src/app/messages/MessageProvider.tsx [hunk 3]
```diff
@@ -7,13 +7,13 @@ import createIntl from './createIntl';
...
 // tslint:disable-next-line:variable-name
-const MessageProvider = (props: { children?: React.ReactNode }) => {
+const MessageProvider = (props: React.PropsWithChildren<{}>) => {
   const services = useServices();
   const [intl, setIntl] = useState<IntlShape | null>(services.intl.current);
   const bookLocale = useSelector(currentLocale);
 
   useEffect(() => {
-    if (!bookLocale) {
+    if (!bookLocale || intl) {
```
Comment (Dantemss) [inline src/app/messages/MessageProvider.tsx:16]: Would this be a problem when switching to a book with a different locale?

**Review by Dantemss — COMMENTED** [review]
I'm not 100% sure so I have some questions.

File: src/app/messages/MessageProvider.tsx (hunk 3 above)
Comment (TomWoodward) [reply src/app/messages/MessageProvider.tsx:16]: Yea, that's not a thing that should ever happen but I can add some logic to test for it

File: src/app/errors/components/OuterErrorBoundary.tsx (hunk 1 above)
Comment (TomWoodward) [reply src/app/errors/components/OuterErrorBoundary.tsx:22]: Yea it shouldn't matter, this is the way it was set up already

**Review by Dantemss — APPROVED** [review]

**Pull request merged**